db/migrate/20230331165559_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key:{to_table: :users}, index: { name: 'likes_authors' }
```

Comment:
- [ ] Good job working on this but please help make sure it references `:author` just like it is in the post migration so that it matches the ERD diagram specifics.